jekyll/_cci2/language-python.md

```diff
@@ -2,104 +2,266 @@
+This key is used
+to issue warnings about breaking changes.
 
-We recommend using a CircleCI pre-built image that comes pre-installed with tools that are useful in a CI environment. You can select the Python version you need from Docker Hub: <https://hub.docker.com/r/circleci/python/>. The demo project uses an official CircleCI image.
+```yaml
+version: 2
+```
+
+### Create a Build Job
 
-Database images for use as a secondary 'service' container are also available on Docker Hub in the `circleci` directory.
+A run is comprised of one or more [jobs]({{ site.baseurl }}/2.0/configuration-reference/#jobs).
+Because this run does not use [workflows]({{ site.baseurl }}/2.0/configuration-reference/#workflows),
+it must have a `build` job.
 
-## Build the Demo Django Project 
+Use the [`working_directory`]({{ site.baseurl }}/2.0/configuration-reference/#job_name) key
+to specify where a job's [`steps`]({{ site.baseurl }}/2.0/configuration-reference/#steps) run.
 
-A good way to start using CircleCI is to build a project yourself. Here's how to build the demo project with your own account:
+```yaml
+version: 2
+jobs:
+  build:  # required for runs that don't use workflows
+    working_directory: ~/circleci-demo-python-django
+```
 
-1. Fork the project on GitHub to your own account.
-2. Go to the [Add Projects](https://circleci.com/add-projects){:rel="nofollow"} page in CircleCI and click the Build Project button next to the project you just forked.
-3. To make changes you can edit the `.circleci/config.yml` file and make a commit. When you push a commit to GitHub, CircleCI will build and test the project.
+### Choose an Executor Type
 
-## Config Walkthrough
+A job's steps occur in a virtual environment called an [executor]({{ site.baseurl }}/2.0/executor-types/).
 
-Start with the version:
+In this example,
+the [`docker`]({{ site.baseurl }}/2.0/configuration-reference/#docker) executor is used
+to specify a custom Docker image.
+The first image listed becomes the job's [primary container]({{ site.baseurl }}/2.0/glossary/#primary-container).
+All of a job's commands execute in this container.
 
 ```yaml
-version: 2 # use CircleCI 2.0
+version: 2
+jobs:
+  build:
+    working_directory: ~/circleci-demo-python-django
+    docker:
+      - image: circleci/python:3.6.4  # primary container
 ```
 
-First, specify a `jobs` key. Each job represents a phase in your Build-Test-Deploy (BTD) process. The sample app only needs a `build` job, so all other configuration will be nested under that key.
+**Note:**
+`circleci/python:3.6.4` is a [convenience image]({{ site.baseurl }}/2.0/circleci-images/) provided by CircleCI.
+These images are extensions of official Docker images
+and include tools useful for CI/CD environments.
+
+### Add Other Services and Set Environment Variables
 
-Specify a working directory and container images for this build in `docker` section:
+You can specify additional containers for services like databases.
```

Comment:
Use active voice, you can -> You must or just specify additional...

Reply:
"You can specify" is active voice; I think what you meant was to remove "can" or make this imperative mood (which I agree with).